lib/dogma/rule/space_after_comma.ex

```diff
@@ -0,0 +1,39 @@
+use Dogma.RuleBuilder
+
+defrule Dogma.Rule.SpaceAfterComma, [spaces: 1] do
+  @moduledoc """
+  A rule that enforces to use `spaces` after comma.
+
+  This rule can be configured with the `spaces` option, which allows you to
+  specify your own number of spaces after comma.
+  """
+  @spaces_pattern ~r/\,(\s*).+/
+  @string_pattern ~r/\"(.*?,.*?)\"/
+
+  def test(rule, script) do
+    script.lines
+    |> Enum.map(&drop_strings/1)
```

Comment:
I think @lpil wanted you to use the `processed_source` instead of the `source` of the Script. It is set via [sigils](https://github.com/lpil/dogma/blob/master/lib/dogma/util/script_sigils.ex) and [strings](https://github.com/lpil/dogma/blob/master/lib/dogma/util/script_strings.ex) helpers and [should not contain any sigils and strings](https://github.com/lpil/dogma/blob/410d88070a511e872d340e88f879a15385962a3e/lib/dogma/script.ex#L89) :+1: